README.md

```diff
@@ -1 +1,67 @@
-# capstone-project-02
\ No newline at end of file
+# Project Title : Capstone Project 2 
+  This is a simple API integration project displaying a list of countries and its information fetching data using API
+  The project is bundled using webpack and makes use of dynamic code through Javascript. the project makes use of jest for unit testing.
+  
+  ![home](https://user-images.githubusercontent.com/51260559/197200034-c2fe80ba-17d8-4de4-b415-f1ac9ed7bd21.png)
+
+
+## Built With
+- CSS
+- HTML
+- JavaScript
+- Bundlled with Webpack
+
+## Live Demo (if available)
+
+## [Live Demo Link](https://arabiu033.github.io/capstone-project-0x02/)
+
+
+## Getting Started
+
+**To use this repositpry you will need to Clone this repository to your machine. A .gitignore file is already created to ignore any file that is irrelevant, especially your node.js file. A linter is already set up to help you check for errors in your codes**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+**Tap on the code option in the repo, then you copy the link to the repo provided**
+### Setup
+**Open your git bash or your IDE terminal then use the git command to clone it to your machine**
+  run : npm run start to start the project on your local machine
+### Install
+**The folder will then be installed in your machine.**
+  To install dependancies, run: npm install 
+### Usage
+**You can edit codes and make changes, dont forget to give credits to me**
+@arabiu033 & @RichardChileya
+### Run tests
+**Linters are already setup in the repo for error checks**
+  if you would like to run jest tests run: npm test
+
+## Authors
+
+👤 **Aminu Rabiu**
+👤 **Richard Chileya**
+
+- GitHub: [@RichardChileya](https://github.com/RichardChileya/)
+- Twitter: [@arabiu033](https://twitter.com/arabiu033)
+- Twitter: [@RichardChileya](https://twitter.com/RichardChileya/)
+- LinkedIn: [Aminu Rabiu](https://www.linkedin.com/in/arabiu033/)
+- LinkedIn: [richard-chileya](https://linkedin.com/in/richard-chileya-1076b4200//)
+
+Contributions, issues, and feature requests are welcome !
+
+Feel free to check the [issues page](https://github.com/arabiu033/capstone-project-0x02/issues).
+
+ **Contributor**
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+## 📝 License
+
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional.
-  Please, consider fixing the broken link to the MIT.md file 🥇

Reply:
We've removed the eslint disables. Thank you.

Reply:
The License has been added , thanks for reminding us